charts/osm/crds/multiclusterservice.yaml

```diff
@@ -41,6 +41,9 @@ spec:
                 - serviceAccount
                 - clusters
               properties:
+                globalIP:
```

Comment:
Would you elaborate why this is needed ? The design doc doesn't talk about this, isn't the IP address of the gateway sufficient ?

Reply:
The envoy listener filter is applied after dns, and uses the IP address of the service to perform the match, since that’s all that’s available on the socket since tls may not have been applied. 

Without an IP address it wouldn’t be able to use it. Correct that it was missed in the initial design doc

Reply:
@steeling could you clarify the following:
1. What is this IP referring to exactly
2. Is this to be used for inbound traffic filtering on the gateway or outbound filtering on the client to access the gateway?

Reply:
1. The IP address for the svc.namespace.cluster.global target
2. inbound filtering on the downstream service (not the gateway), to pick the outbound cluster

Reply:
resolved based on in person conversation